### PR TITLE
fix: spec reading where asset is optional

### PR DESCRIPTION
**Pull request by deryrahman**
Change logs:
- [x] add capability to read job without assets folder

### CODE REVIEW

**Review by irainia — CHANGES_REQUESTED** [review]
we can have further discussion if required

File: client/local/internal/internal.go [hunk 1]
```diff
@@ -13,6 +13,17 @@ import (
 	"github.com/odpf/optimus/client/local"
 )
 
+func PathExist(specFS afero.Fs, path string) (bool, error) {
```
[the remark this answers is not in the document]
Comment (deryrahman) [reply client/local/internal/internal.go:16]: It's intended to check if the path is exist, regardless it's folder or file. We use this to check folder assets

[the remark this answers is not in the document]
Comment (irainia) [reply client/local/internal/internal.go:16]: Aren't we checking whether the file or directory exists? The `path` is there, the one we send through the parameter.

File: client/local/internal/internal.go [hunk 2]
```diff
@@ -13,6 +13,17 @@ import (
...
+	_, err := specFS.Stat(path)
+	if err == nil {
+		return true, nil
+	}
+	if os.IsNotExist(err) {
```
Comment (irainia) [inline client/local/internal/internal.go:21]: On this, please use `errors.Is(...)` as this one is recommended by Go [here](https://pkg.go.dev/os#IsNotExist).

**Review by irainia — CHANGES_REQUESTED** [review]

File: client/local/internal/internal.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (deryrahman) [reply client/local/internal/internal.go]: as per discussion, we remove the func and use inline instead

**Review by irainia — APPROVED** [review]

**Pull request merged**